homework.js

```diff
@@ -0,0 +1,32 @@
+Array.prototype.random = function() {
+  return this[Math.floor(Math.random() * this.length)];
+};
+
+function Fabrica(capacity = 32) {
+  this.capacity = capacity;
+  this.products = [];
+}
+
+Fabrica.prototype.adauga = function(product = {}) {
```

Comment:
As aprecia daca am vorbi toti aceeasi limba atunci cand vine vorba de programare, iar cum totul este in engleza, si metodele tale ar trebui sa fie scrise tot in engleza :)
```suggestion
MyFactory.prototype.add = function(product = {}) {
```